eZ/Publish/API/Repository/Values/User/Role.php

```diff
@@ -38,6 +55,41 @@
     protected $identifier;
 
     /**
+     * The status of the role.
+     *
+     * @var int One of Role::STATUS_DEFINED|Role::STATUS_DRAFT|Role::STATUS_MODIFIED
+     */
+    protected $status;
```

Comment:
ok, let's keep it like this.
Just a bit strange as you probably won't get published objects unless defined.